config/policy.go

```diff
@@ -604,10 +604,11 @@ func (p *Policy) Validate() error {
 			return fmt.Errorf("config: couldn't decode custom ca: %w", err)
 		}
 	} else if p.TLSCustomCAFile != "" {
-		_, err := os.Stat(p.TLSCustomCAFile)
+		ca, err := os.ReadFile(p.TLSCustomCAFile)
 		if err != nil {
 			return fmt.Errorf("config: couldn't load client ca file: %w", err)
 		}
+		p.TLSCustomCA = base64.StdEncoding.EncodeToString(ca)
```

Comment:
I know we already do it, so it's fine, but I really dislike that `Validate` has side effects.

Reply:
yeah, I don't like this either, but it fits with the other code in this function. :shrug: